client/my-sites/post-share/index.jsx

```diff
@@ -262,6 +301,31 @@ class PostShare extends Component {
 			translate,
 		} = this.props;
 
+		if ( this.props.scheduling ) {
+			return (
+				<Notice status="is-warning" showDismiss={ false }>
+					{ translate( 'We are writing your shares to the calendar…' ) }
+				</Notice>
+			);
+		}
+		if ( this.props.scheduledAt ) {
+			return (
+				<Notice status="is-success" onDismissClick={ this.dismiss }>
+					{ translate( 'We`ll share your post on %s.', {
```

Comment:
I noticed this via a P2 screenshot. @artpi, could we use an actual apostrophe instead of the backtick?

```diff
-{ translate( 'We`ll share your post on %s.', {
+{ translate( "We'll share your post on %s.", {
```